### ISSUE TITLE
Obsolete warnings in android gradle

### ISSUE THREAD

**Issue by harshapulikollu** [flutter/flutter#28318]
Updated to `gradle version 3.3.1 ` -- `classpath 'com.android.tools.build:gradle:3.3.1'` and I'm getting this warning..
```
WARNING: API 'variant.getMergeAssets()' is obsolete and has been replaced with 'variant.getMergeAssetsProvider()'.
It will be removed at the end of 2019.

WARNING: API 'variantOutput.getProcessResources()' is obsolete and has been replaced with 'variantOutput.getProcessResourcesProvider()'.
It will be removed at the end of 2019.
```

**Comment by zoechi** [comment]
Seems to be coming from a plugin, but I don't think this is enough information to track down the root cause.

**Comment by harshapulikollu** [comment]
This is the whole message 

```
WARNING: API 'variantOutput.getProcessResources()' is obsolete and has been replaced with 'variantOutput.getProcessResourcesProvider()'.
It will be removed at the end of 2019.
For more information, see https://d.android.com/r/tools/task-configuration-avoidance.
To determine what is calling variantOutput.getProcessResources(), use -Pandroid.debug.obsoleteApi=true on the command line to display a stack trace.
Affected Modules: app
```
I tired to run command `-Pandroid.debug.obsoleteApi=true` on command for stacktrace but I got error
```
'Pandroid.debug.obsoleteApi' is not recognized as an internal or external command,
operable program or batch file.

```

**Comment by rmtmckenzie** [comment]
I see the `variantOutput.getProcessResources()` part of this too although not the other part. In my case it's coming from flutter.gradle line 446:
```
                variant.outputs[0].processResources.dependsOn(copyFlutterAssetsTask)
```

This appears to have previously been fixed (see #26544) but then reverted for some reason.

**Comment by BondarenkoStas** [comment]
Closing as duplicate of #26544
